data-models/common/climate/schema.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://akenza.github.io/device-types-library/data-models/common/climate/schema.json",
+  "$defs": {
+    "temperatureC": {
```

```suggestion
    "temperatureCelsius": {
```

As already mentioned in the `README` part, I'd write our the unit here. Same applies to all following units for temperature.